### PR TITLE
new LC application

### PR DESCRIPTION
**Pull request by adamdruppe**

### CODE REVIEW

File: app/assets/javascripts/public/enrollments.js [hunk 1]
```diff
@@ -38,7 +37,7 @@ $(document).ready(function() {
     
     // Generate TOC:
     $('h2').not('.sr-only, #form-almost-done h2').each(function() {
-      $('#jumplinks').append('<li class="page-jump '+$(this).closest('div').prop('class')+'"><a href="#'+$(this).closest('div').prop('id')+'"><div class="jump-icon"></div>'+$(this).text()+'</a></li>');
```
Comment (sadleb) [inline app/assets/javascripts/public/enrollments.js]: I don't know what this is doing, but I'll trust it's better.

File: app/views/enrollments/_form.html.erb [hunk 2]
```diff
@@ -414,20 +410,38 @@
...
+
+            <div class="form-group coach">
+              <%= f.check_box :affirm_commit_coach %>
+              <%= f.label :affirm_commit_coach, '
+                I affirm that to the best of my knowledge I can fulfill the commitment to participate in all Leadership Coach activities, which can include: facilitating weekly cohort sessions (a mix of in-person and virtual); leading virtual individual coaching sessions participants; and delivering feedback on selected artifacts (e.g., resumes) submitted by participants.  ', 'style' => 'display: inline' %>
+            </div>
+
+            <div class="form-group coach">
+              <span class="coach">If any, please let us know any weeks or weekends you know you will not be available during the spring (2015).</span>
+              <%= f.text_area :cannot_attend, 'class' => 'form-control', 'maxlength' => '300' %>
```
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: Is this mapped into Salesforce?  If not, it's probably an oversight.

File: app/views/enrollments/_form.html.erb [hunk 3]
```diff
@@ -437,30 +451,32 @@
...
               <div class="form-group">
                 <label>Anything else you'd like to share?</label>
-                <p class="help">Please let us know any additional information you wish to share about your interest in <%= @program_title %>.</p>
+                <p class="help">Please let us know any additional information you wish to share about your interest in <%= @program_title %> - <%= @program_site %>.</p>
                 <%= f.text_area :comments, 'class' => 'form-control', 'maxlength' => '300' %>
               </div>
+
+              <p>Finally, please help us to continuously improve.</p>
+
+              <% if !@sourcing_options.empty? %>
```
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: Can you add a comment with an example of the format this expects for sourcing options?

**Comment by sadleb** [comment]
You checked off that you renamed lead_sources to sourcing_info, but that doesn't seem to be in this commit.

**Comment by adamdruppe** [comment]
On Thu, Apr 09, 2015 at 09:55:33AM -0700, Brian Sadler wrote:
> You checked off that you renamed lead_sources to sourcing_info, but that doesn't seem to be in this commit.

I did that one yesterday, I think it is already in staging.

Line 255 of the enrollment.rb shows it.

**Comment by sadleb** [comment]
> I did that one yesterday, I think it is already in staging.
> 
> Line 255 of the enrollment.rb shows it.

I'm talking about L143 of enrollement_controller.rb

**Comment by adamdruppe** [comment]
On Thu, Apr 09, 2015 at 11:52:17AM -0700, Brian Sadler wrote:
> I'm talking about L143 of enrollement_controller.rb

That's just the internal name, changing those is going
to require a big database migration and I plan to do
them as part of a future pass (already in asana I believe,
rename those and remove the obsolete ones).

**Pull request merged**